nnpdf_data/nnpdf_data/new_commondata/NNPDF_POS_2P24GEV/metadata.yaml

```diff
@@ -559,7 +559,7 @@ implemented_observables:
     description: Deep Inelastic Scattering
     label: 'positivity dataset: DIS light quark longitudinal structure function $F_L^l$'
     units: ''
-  process_type: POS_FLL
+  process_type: POS_DIS
   tables: []
```

Comment:
Hi @RoyStegeman, I think it would be good to change the `kinematic_coverage` as well as `kinematics` for `POS_FLL` as it was done for the other `POS_F2` sets so as to avoid any bug in processes